### PR TITLE
Bump CRO to v0.8.2

### PR DESCRIPTION
**Pull request by davidkirwan**
## Overview
Release v0.8.2 to include: https://github.com/integr8ly/cloud-resource-operator/pull/105

### CODE REVIEW

File: deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    type: AllNamespaces
+  keywords:
+  - integreatly
+  maintainers:
+  - email: integreatly-support@redhat.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Integreatly
+  replaces: cloud-resources.v0.7.1
```
Comment (ciaranRoche) [inline deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml]: is this replace version correct?

Comment (davidkirwan) [reply deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml]: So this is the latest version in the integreatly operator thats on master, however on CRO master, its `v0.8.1`? Should I leave it at `v0.8.1` here in the PR, but manually change it in the integreatly operator?

Comment (ciaranRoche) [reply deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml]: This requires a wider conversation on how we map CRO releases to the Integreatly operator. I would lean toward we stick with what ever is the current release of CRO, and we update this manually in the integreatly operator.

Comment (davidkirwan) [reply deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml]: Yeah.. I'd agree there, let me change it to whats in CRO master. `v0.8.1`

**Pull request merged**